internal/data/spicedb.go

```diff
@@ -242,6 +244,49 @@ func (s *SpiceDbRepository) LookupResources(ctx context.Context, resouce_type *a
 	return resources, errs, nil
 }
 
+func (s *SpiceDbRepository) ImportBulkTuples(stream grpc.ClientStreamingServer[apiV1beta1.ImportBulkTuplesRequest, apiV1beta1.ImportBulkTuplesResponse]) error {
+	if err := s.initialize(); err != nil {
+		return err
+	}
+
+	var totalImported uint64
+	for {
+		req, err := stream.Recv()
+		if err != nil {
+			if !errors.Is(err, io.EOF) {
+				if err := stream.SendAndClose(&apiV1beta1.ImportBulkTuplesResponse{NumImported: totalImported}); err != nil {
+					return status.Errorf(codes.Internal, "failed to send response: %v", err)
+				}
+			}
+			return err
+		}
+		inputRelationships := (*req).Tuples
+		batch := []*v1.Relationship{}
+		for _, tuple := range inputRelationships {
+			batch = append(batch, createSpiceDbRelationship(tuple))
```

Comment:
```suggestion
			tuple.Relation = addRelationPrefix(tuple.Relation, relationPrefix)
			batch = append(batch, createSpiceDbRelationship(tuple))
```
When adding a relationship we need to make sure that we prefix the relations before creating them. You shouldn't have to send requests with "t_member" yourself